packages/shared_preferences/shared_preferences_linux/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ environment:
   flutter: ">=1.12.8 <2.0.0"
 
 dependencies:
-  file: ^5.1.0
+  file: ">=5.1.0 < 7.0.0"
```

Comment:
I assume this is <7 rather than something like ^6.0.0 because of the use of a dev version on the 6.0.0?

Reply:
yep